src/app/home/home.page.ts

```diff
@@ -74,4 +79,57 @@ export class HomePage {
       this.utilService.clearDatabase();
     }
   }
+
+  async getProfileDetails() {
+  await this.loader.showLoading("LOADER_MSG")
+    this.profileService.getProfileAndEntityConfigData().subscribe(async (mappedIds) => {
+      if (mappedIds) {
+        this.profilePayload = mappedIds;
+        this.getdata();
+      }
+    });
+  }
+
+  getdata() {
+    const urls = {
```

Comment:
if search and filter feature are not present in this listing pages, please remove it

Reply:
done